src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,110 @@
+
+## Before You Begin
+
+Before diving into the installation and usage of Daytona, it is crucial to ensure that your system meets the necessary prerequisites and that you have all the required tools and configurations in place. This section outlines the key requirements and preparatory steps you should take before getting started with Daytona.
+
+### System requirements
+
+Daytona supports multiple operating systems and architectures. Ensure your system matches one the following specifications:
+
+| **Operating Systems**                         | **Architectures**                                   |
+|-----------------------------------------------| ----------------------------------------------------|
+| [Linux](installation/installation#linux)      | `x86_64` or `AArch64`                               |
+| [macOS](/installation/installation#macos)     | `x86_64 (Intel-based)` or `AArch64 (Apple Silicon)` |
+| [Windows](/installation/installation#windows) | `x86_64` or `AArch64`                               |
+
+### Dependencies
+
+Daytona relies on specific dependencies to function correctly. Ensure that these dependencies are installed and properly configured on your system:
+
+| **Dependency**            | **Description**                                                                                 |
+|---------------------------|-------------------------------------------------------------------------------------------------|
+| Docker                    | Daytona uses Docker to create and manage isolated development environments.                     |
+| Git                       | Daytona requires Git to clone repositories and manage version control operations.               |
+| Administrator Privileges  | Ensure that you have the necessary permissions to install and configure Daytona on your system. |
+
+### Tools
+
+Certain tools are essential for effectively using Daytona. Below is a list of tools that will facilitate interaction and development within the Daytona environment:
+
+| **Tool**                          | **Description**                                                                                                                                                    |
+|-----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Command-Line Interface (CLI)      | Essential for executing Daytona commands and interacting with the system.                                                                                          |
```

Comment:
@stefanicjuraj CLI and ssh i think are also things that are unnecessary . IDE, we can mention that daytona come with a cloud ide embedded but you can use you local ides if you like

Reply:
CLI is a bit redundant, I suggest we remove that.

The SSH Client is a requirement because we use SSH to connect to any project (even if it's local)